crates/cache/src/config.rs

```diff
@@ -301,6 +301,10 @@ impl CacheConfig {
     generate_setting_getter!(file_count_limit_percent_if_deleting: u8);
     generate_setting_getter!(files_total_size_limit_percent_if_deleting: u8);
 
+    pub fn builder() -> CacheConfigBuilder {
```

Comment:
Mind adding some docs to this method since it'll be public-facing in the `wasmtime` crate API?